pandas/tests/reshape/merge/test_merge.py

```diff
@@ -39,6 +39,42 @@ def get_test_data(ngroups=NGROUPS, n=N):
     return arr
 
 
+def get_series():
+    return [
+        pd.Series([1], dtype='int64'),
+        pd.Series([1], dtype='Int64'),
+        pd.Series([1.23]),
+        pd.Series(['foo']),
+        pd.Series([True]),
+        pd.Series([pd.Timestamp('2018-01-01')]),
+        pd.Series([pd.Timestamp('2018-01-01', tz='US/Eastern')]),
+    ]
+
+
+def get_series_nan():
+    return [
+        pd.Series([np.nan], dtype='Int64'),
+        pd.Series([np.nan], dtype='float'),
+        pd.Series([np.nan], dtype='object'),
+        pd.Series([pd.NaT]),
+    ]
+
+
+@pytest.fixture(params=get_series(), ids=lambda x: x.dtype.name)
```

Comment:
can you add a 1-line comment to each of these, and make the names a bit more verbose; its easy to see this in the diff, but when looking at the raw column these names are hard to grok.

Reply:
Are you talking about column names `a` and `b` or fixture names `value_col` and `value_col2`?

Reply:
the fixture names

Reply:
done